### ISSUE TITLE
CheckBox: High contrast issues

### ISSUE THREAD

**Issue by mikewheaton** [OfficeDev/office-ui-fabric-core#632]
1. Checkboxes need a hover and a focus state in high contrast mode.
2. Disabled states aren't indicated correctly. Needs to be green in HC black, and maroon in HC White.
 - ![image](https://cloud.githubusercontent.com/assets/1382445/15807690/94e16ff2-2b19-11e6-98ff-c0b6c0295f28.png)

**Comment by mikewheaton** [comment]
This issue was moved to OfficeDev/office-ui-fabric-js#53
